packages/react/src/helpers/__tests__/validateEircode.test.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  type AddressSchemaAddressType,
+  type AddressSchemaLineType,
+  getAddressSchema,
+} from '@farfetch/blackout-client';
+import { validateEircode } from '../index.js';
+
+jest.mock('@farfetch/blackout-client', () => ({
+  getAddressSchema: jest.fn(),
+}));
+
+const mockGetAddressSchema = jest.mocked(getAddressSchema);
+const eircodeRegex =
+  '^(?=.{7,8}$)^([AC-FHKNPRTV-Y]{1}[0-9]{2}|D6W)[ ]?[0-9AC-FHKNPRTV-Y]{4}$';
+
+describe('validateEirocde', () => {
+  beforeEach(() => {
+    mockGetAddressSchema.mockResolvedValueOnce([
```

Comment:
I believe you don't need to do this on `beforeEach`, you can set the value right on the mock (line 9) or at least have the `mockResolvedValue` outside 🤔